### ISSUE TITLE
Focus on panel not working 

### ISSUE THREAD

**Issue by gjyotin305** [warpdotdev/Warp#4298]
### Describe the bug

The shift focus panel is not working for my linux setup.

Setup : Arch Linux :  6.7.5-arch1-1
Terminal : zsh

### To reproduce

Steps to reproduce:

1. Open a panel (Ctrl  + Shift + D)
2. Try to focus but it won't ( Ctrl + Shift + (   or Ctrl + Shift + ) )

### Expected behavior

It should change focus with the given or even by changing the keybindings.

### Operating system

Linux

### Operating system and version

Arch Linux

### Shell Version

zsh 5.9

### Current Warp version

v0.2024.02.20.08.01.stable_01

### Regression

No, this bug or issue has existed throughout my experience using Warp

### Recent working Warp date

N.A

### Does this block you from using Warp daily?

Yes, this issue prevents me from using Warp daily.

### Is this a Warp specific issue? (i.e. does it happen in Terminal, iTerm, Kitty, etc.)

Yes, this I confirmed this only happens in Warp, not other terminals.

### Warp Internal (ignore): linear-label:b9d78064-c89e-4973-b153-5178a31ee54e

None

**Comment by LFd3v** [comment]
I do not know if this was a problem with the displayed help text, but here it seems to work if I use square brackets instead of parenthesis (I did not change the default keybindings).

**Comment by gjyotin305** [comment]
> I do not know if this was a problem with the displayed help text, but here it seems to work if I use square brackets instead of parenthesis (I did not change the default keybindings).

Yeah it works but smh, the keybindings they have given as default doesnt seem to work. Even if I change the keybindings, then also it doesnt work.It needs to be fixed

**Comment by dheater** [comment]
I'm also finding that ctrl-alt-left/right are not working. up and down are working

**Comment by venpopov** [comment]
Same issue on MacOS. `cmd+[` works to switch focus between panels, which I found by chance, but the default bindings, which I haven't changed, do not work.

**Comment by munoztd0** [comment]
same here
